src/content/docs/builders/node-operators/skale-nodes.mdx

```diff
@@ -1,28 +1,28 @@
 ---
-title: SKALE Nodes
-description: Intro to SKALE Nodes
+title: SKALE Supernodes
+description: Intro to SKALE Supernodes
 ---
 import { Tabs, TabItem } from '@astrojs/starlight/components';
 import ToolLayout from "../../../../components/Tool/ToolLayout.astro";
 
-The SKALE Network consists of a large pool of validator nodes, all running concurrently and independently, validating transactions within the SKALE Chains they oversee. These nodes coordinate with a set of smart contracts that run on the Ethereum mainnet called SKALE Manager.
+The SKALE Network consists of a large pool of validator supernodes, all running concurrently and independently, validating transactions within the SKALE Chains they oversee. These supernodes coordinate with a set of smart contracts that run on the Ethereum mainnet called SKALE Manager.
 
-Each SKALE Chain comprises a collective of randomly appointed virtualized subnodes that run a containerized SKALE EVM-client daemon (skaled) and SKALE ABBA-based consensus algorithm. Nodes in the SKALE Network aren’t restricted to operating a single chain but instead can support multiple chains via virtualized subnodes. This multiplex capability is made possible via a containerized subnode architecture deployed on each node in the network. Each node is virtualized and can participate as a validator via this subnode architecture for an independent number of SKALE Chains.
+Each SKALE Chain comprises a collective of randomly appointed virtualized nodes that run a containerized SKALE EVM-client daemon (skaled) and SKALE ABBA-based consensus algorithm. Supernodes in the SKALE Network aren’t restricted to operating a single chain but instead can support multiple chains via virtualized nodes. This multiplex capability is made possible via a containerized node architecture deployed on each supernode in the network. Each supernode is virtualized and can participate as a validator via this node architecture for an independent number of SKALE Chains.
```

Comment:
Remove virtualized in first sentence

Reply:
In the second sentence change virtualized to containerized. 

E.g support multiple chains via containerized nodes

Reply:
The last sentence should be node node supernode